download/upgradeRecipe/upgradeRecipe8.adoc

```diff
@@ -0,0 +1,53 @@
+= Upgrade recipe 8
+:awestruct-description: Upgrade to OptaPlanner 8 from a previous version.
+:awestruct-layout: upgradeRecipeBase
+:awestruct-priority: 0.5
+:awestruct-upgrade_recipe_version: 8
+
+
+// TODO Once 8.0.0.Beta1 is out, remove these lines and add a link to this page in ./index.adoc
+*OptaPlanner 8 is in _alpha_ development.
+There is no use in waiting for it.
+Please use and download the latest 7.x release instead.*
+
+== From 7.39.0.Final to 8.0.0.Beta1
```

Comment:
It's too early for this.. I'd rather not update this every 3 weeks.

Reply:
I wanted to state "7.x", but if they're at 7.15, they should first follow the upgradeRecipe7.
How about "From 7.39.0.Final or higher to 8.0.0.Beta1"?

Note that this page isn't linked anywhere _yet_ in the rest of the optaplanner.org website

Reply:
OK. "7.39.0.Final or higher".